doc/manual/manual.tex

```diff
@@ -969,6 +969,31 @@ \subsubsection{Coefficient self-consistency}
   compressibility for each of the different approximations. The conversion between isothermal
   and isentropic compressibilities is given in ~\eqref{eq:mm_isentropic_compressibility}.}
 
+
+\subsubsection{Coefficient averaging}
+In multiphase rocks, or multirock areas in convection simulations, properties must be averaged.
+For viscosity and conductivity, there is no single expression that describes how averaging should
+be performed; indeed, these properties are dependent on rock texture and mineral alignment,
+both of which may change through time as strain accumulates, and chemical diffusion and
+reactions take place.
+
+In the case of density, thermal expansivity, heat capacity and bulk compressibility, there is
+one correct way of averaging. Here we must consider conservation of mass and composition in a
+multicomponent rock $r$. If component $i$ has masses $M_i$ and densities $\rho_i$, we
```

Comment:
do you mean "fractional masses"? From the formula I believe you mean that they should add up to one, right?

Reply:
No, I mean masses. x_i are the fractional masses (in the second of the following two equations). The division by the total mass is trivial, so I can remove it if you want. I just find this way marginally easier to follow.